src/utils/data.js

```diff
@@ -425,25 +441,40 @@ export class DataUtil {
     };
   };
 
-  getStandardDevData = () => {
-    const { averageGlucose, bgData, total } = this.getAverageGlucoseData(true);
+  getStandardDevData = ( cvByDate = false) => {
+    const { bgData, averageGlucose, bgDataByDate, total } = this.getGlucoseDataByDate(true);
 
-    if (bgData.length < 3) {
+    if (total < 3) {
       return {
         averageGlucose,
         insufficientData: true,
-        standardDeviation: NaN,
         total,
+        standardDeviation: NaN,
       };
     }
-
+    
     const squaredDiffs = _.map(bgData, d => (d.value - averageGlucose) ** 2);
     const standardDeviation = Math.sqrt(_.sum(squaredDiffs) / (bgData.length - 1));
 
+    const coefficientOfVariationByDate = {};
+    if (cvByDate) {
+      _.forEach(bgDataByDate, function(value, key) {
+        // ignore days having less than 3 glycemia values
+        if (value.length >= 3 ) {
+          const avgGlucose =  _.meanBy(value, 'value');
+          const squaredDiffs = _.map(value, d => (d.value - avgGlucose) ** 2);
+          const standardDeviation = Math.sqrt(_.sum(squaredDiffs) / (value.length - 1));
+          coefficientOfVariationByDate[key] = standardDeviation / avgGlucose * 100;
```

Comment:
Comments are needed on this formula.
The math formula is needed for me, at least in the Jira
Something like:
![image](https://user-images.githubusercontent.com/53182545/89430879-84413980-d73f-11ea-86c4-0a62585e7b6e.png)

Reply:
JIRA updated